### DISCUSSION TITLE
Welcome to tdep-tutorials Discussions!

### DISCUSSION THREAD

**Discussion by flokno** [tdep-developers/tdep-tutorials#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ganesh028** [comment]
Hello Flokno,
I am wondering how I could get rid of this symmetry error.

Best regards,
Dr. Ganesh K Nayak

... reading unitcell
 ... min cutoff: 2.68274
 ... max cutoff: 5.60156
 
 DETERMINING SYMMETRY OF INTERACTION TENSORS
 ... constructed distance tables (0.01014s)
 ... built singlets
 ... built pairs
 ... built prototype tuplets (0.00019s)
 ... expanded symmetry operations (0.10744s)
  ... reducing pairs                     100.0% |========================================|  0.00046s
 
 ERROR
 exit code 4: symmetry error
 
 Could not locate tuplet
 
     occurs in file: lo_symmetry_of_interactions_tuplets.f90
     occurs on line: 730
        active rank: 0 out of 1

**Reply by flokno** [reply]
Hi @ganesh028 👋 , can you open an [issue](https://github.com/tdep-developers/tdep/issues) and include your input files there?

**Reply by ganesh028** [reply]
Thanks, Flikno, I just did. Thank you in advance!
